egs2/ljspeech/tts1/run.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+# Set bash to  'debug' mode, it will exit on:
+# -e 'error', -u 'undefined variable', -o ... 'error in pipeline', -x 'print commands',
+
+set -e 
+set -u
+set -o pipefail
+
+ngpu=$1
+
+#tts_config=conf/train_transformer.v3.yaml
+tts_config=conf/train_fastspeech.v3.yaml
+decode_config=conf/decode.yaml
+
+teacher_model_path=exp/tts_train_tacotron2.v3_fbank/valid.loss.best.pth
+teacher_model_config=exp/tts_train_tacotron2.v3_fbank/config.yaml
+
+./tts.sh \
+  --ngpu ${ngpu} \
+  --audio_format wav \
+  --feats_type fbank \
```

Comment:
Would you change just as following:
https://github.com/espnet/espnet/blob/23d06901f388888f7608979ba3b1a287d749c4f0/egs2/jsut/asr1/run.sh#L10-L15

If sampling frequency is changed from the original rate, we need to recreate the audio files, so please use flac in that case to reduce disk space.